src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -61,31 +66,10 @@ export function getPsvTechRecord(dtpNumbersFromRefData: FormNodeOption<string>[]
             type: FormNodeTypes.CONTROL,
             editType: FormNodeEditTypes.AUTOCOMPLETE,
             options: dtpNumbersFromRefData,
-            validators: [{ name: ValidatorNames.Required }]
+            validators: [{ name: ValidatorNames.Required }],
```

Comment:
Does this not also need maxLength of 6?

Reply:
good spot, will push up

Reply:
there is existing validation where if the dtpNumber isn't found in the ref data, then you will get back a 'DTP Number is invalid' error